server/game/cards/12-2T2D/Bedazzle.js

```diff
@@ -0,0 +1,40 @@
+const SpellCard = require('../../spellcard.js');
+
+class Bedazzle extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Bedazzle',
+            playType: 'cheatin resolution',
+            cost: ability.costs.bootSelf(),
+            difficulty: 3,
+            choosePlayer: true,
+            onSuccess: (context) => {
+                context.chosenPlayer.modifyRank(-2, context);
+                this.game.addMessage('{0} uses {1} to lower {2}\'s draw hand by 2 ranks; Current rank is {3}',
+                    context.player, this, context.chosenPlayer, context.chosenPlayer.getTotalRank());
+                if(this.game.shootout) {
+                    context.ability.selectAnotherTarget(context.player, context, {
+
+                        activePromptTitle: 'Choose opposing dude to bedazzle',
+                        waitingPromptTitle: 'Waiting for opponent to select dude',
+                        cardCondition: card => card.location === 'play area' &&
+                            card.controller === context.player.getOpponent() &&
+                            card.isParticipating(),
+                        cardType: 'dude',
+                        onSelect: (p, dazzlee) => {
+                            const dazzleAmount = Math.max(dazzlee.bullets, 0);

```

Comment:
This is not needed as basecard has a `bullets` getter that always returns number higher than 0.

Reply:
It took a while parsing dudecard vs drawcard vs basecard, but ok.